.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "Feature request \U0001F30D"
+about: Contribute to eo-learn
```

Comment:
Perhaps this is more appropriate if this were a PR. I would suggest "What is missing?" or something. Otherwise, the templates look good and you can merge them. I would suggest using similar templates in sh-py.